### ISSUE TITLE
How to install the Hardware-Resource-Discovery

### ISSUE THREAD

**Issue by DuShaohe** [NIVeriStandAdd-Ons/Hardware-Resource-Discovery#1]

**Comment by DuShaohe** [comment]
I have download it，but I can‘t find the setup installer

**Comment by pokemane** [comment]
Yeah there is no .lvproj and therefore no build spec.  @eatondan was that intended?

@DuShaohe in the meantime, make a new folder `.../vi.lib/NI/Hardware Resource Discovery` and copy-paste everything under the `Source` dir in this project to that new folder.  That seems to be the extent of what the other similarly-structured CD projects with build specs or .vip's do.
